src/hotspot/share/utilities/elfFile.cpp

```diff
@@ -1624,6 +1628,29 @@ bool DwarfFile::LineNumberProgram::get_filename_from_header(const uint32_t file_
   return false;
 }
 
+// Remove everything before the last slash including the slash itself to get the actual filename. This is required, for
+// example, for Clang debug builds which emit a relative path while GCC only emits the filename.
+void DwarfFile::LineNumberProgram::strip_path_prefix(char* filename, const size_t filename_len) {
+  char* last_slash = strrchr(filename, *os::file_separator());
+  if (last_slash != nullptr) {
+    uint16_t index_after_slash = (uint16_t)(last_slash + 1 - filename);
```

Comment:
Why uint16_t? 

We have `pointer_delta()` for that btw if you want to be super correct. See globalDefinitions.hpp

Reply:
> Why uint16_t?

There is no specific reason. I'll change it to `uint32_t`. 
 
> We have `pointer_delta()` for that btw if you want to be super correct. See globalDefinitions.hpp

Ah that's great! Will use that one instead.